assets/stylesheets/components/_global-header.scss

```diff
@@ -4,7 +4,11 @@
 
 .global-header--inner {
   @extend %content-block;
-  padding: 24px;
+  padding: ($baseline-grid-unit * 4);
```

Comment:
Yep, agreed. I'd drop this change for now though as it's already been merged with #27. 

Maybe add the media query change to the top/bottom padding with this though.